shared/util/codeql/suppression/AlertSuppression.qll

```diff
@@ -0,0 +1,112 @@
+/** Provides a module for constructing alert suppression queries. */
+signature module InputSig {
+  class SingleLineComment {
+    string toString();
+
+    predicate hasLocationInfo(
+      string filepath, int startline, int startcolumn, int endline, int endcolumn
+    );
+
+    string getText();
+  }
+}
```

Comment:
```suggestion
/** A single line comment, used as a signature for alert suppression queries. */
signature class SingleLineCommentSig {
  string toString();

  predicate hasLocationInfo(
    string filepath, int startline, int startcolumn, int endline, int endcolumn
  );

  string getText();
}
```

?

Reply:
The follow-up pull request adds another class to the signature: https://github.com/github/codeql/pull/11723

Reply:
You could add `AstNode` as a second parameter to the shared module?  
Especially since there is no inheritance/dependencies between `AstNode` and `SingleLineComment`.